src/git/run.rs

```diff
@@ -210,11 +256,13 @@ impl GitRunInfo {
         repo: &Repo,
         event_tx_id: Option<EventTransactionId>,
         args: &[S],
-    ) -> eyre::Result<Vec<u8>> {
+        opts: Option<GitRunOpts>,
```

Comment:
I think it's better to force `opts: GitRunOpts` here and have the user pass `Default::default` where they intend to use the default options.

Reply:
Fixed 👍